src/test/java/seedu/address/logic/commands/AddEventCommandTest.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND;
+import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalProjects.getTypicalProjectsFolder;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Interval;
+import seedu.address.model.task.repeatable.Event;
+import seedu.address.testutil.EventBuilder;
+
+public class AddEventCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalProjectsFolder(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() throws Exception {
+        Event validEvent = new EventBuilder().withDescription("CS2106 Tutorial")
+                .withAtDate(LocalDate.of(2020, 01, 01)).withInterval(Interval.WEEKLY).build();
+
+        CommandResult commandResult = new AddEventCommand(INDEX_FIRST, validEvent).execute(model);
+
+        assertEquals(String.format(Messages.MESSAGE_ADD_EVENT_SUCCESS, validEvent), commandResult.getFeedbackToUser());
+    }
+
+    @Test
+    public void execute_invalidProjectIndex_throwsCommandException() {
+        Event eventToAdd = new EventBuilder().withDescription("CS2106 Tutorial")
+                .withAtDate(LocalDate.of(2020, 01, 01)).withInterval(Interval.WEEKLY).build();
+        // Typical project list contains only 2 projects
+        AddEventCommand addEventCommand = new AddEventCommand(INDEX_THIRD, eventToAdd);
+
+        assertThrows(
+                CommandException.class,
+                Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX, () -> addEventCommand.execute(model)
+        );
+    }
+
+    @Test
+    public void execute_duplicatePerson_throwsCommandException() {
```

Comment:
The method header name here is wrong. For other methods, your method header names also seem irrelevant to the what the test actually does.

Reply:
resolved